src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordClone.cs

```diff
@@ -104,9 +102,9 @@ internal override ImmutableArray<string> GetAppliedConditionalSymbols()
         internal override IEnumerable<SecurityAttribute> GetSecurityInformation()
             => Array.Empty<SecurityAttribute>();
 
-        internal override bool IsMetadataNewSlot(bool ignoreInterfaceImplementationChanges = false) => false;
+        internal override bool IsMetadataNewSlot(bool ignoreInterfaceImplementationChanges = false) => true;
```

Comment:
>internal override bool IsMetadataNewSlot(bool ignoreInterfaceImplementationChanges = false) => true; [](start = 8, length = 100)

It looks like meaningful changes in this file a obsolete after [https://github.com/dotnet/roslyn/pull/44852](https://github.com/dotnet/roslyn/pull/44852)